Library/Formula/meanwhile.rb

```diff
@@ -0,0 +1,299 @@
+class Meanwhile < Formula
+  desc "Implementation of Lotus Sametime Community Client protocol"
+  homepage "http://meanwhile.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/meanwhile/meanwhile/1.0.2/meanwhile-1.0.2.tar.gz"
+  sha256 "38a8059eaa549cbcb45ca0a5b453e9608ceadab360db2ae14581fb22ddabdf23"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  patch :DATA
```

Comment:
I've added a comment explaining the patch which uses only the glib.h. Unfortunately, the meanwhile project page has not been updated [since 2006](http://meanwhile.sourceforge.net/) and there has [not been response for how to submit patches](http://sourceforge.net/p/meanwhile/discussion/378953/thread/ca0f1d17/?limit=25#9a38).